### PR TITLE
Title: Hello Microverse project

### PR DESCRIPTION
**Pull request by Lizdev-05**
In this milestone, I did the following:
1. Add the index.html file
2. Add style.css file
3. Correct the linter errors.

### CODE REVIEW

File: .githubworkflows/.githubworkflowslinters.yml.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
```
Comment (danmainah) [inline .githubworkflows/.githubworkflowslinters.yml.txt]: - [ ] Kindly make sure to add the correct files for the linter set up, you have added the wrong file. Also, make sure all the checks are passing.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+about.html
```
Comment (danmainah) [inline .gitignore]: - [ ] Kindly remove the `about.html` from your .gitignore file, also add `test.md` and `node modules` directory in your ignored files.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Lizdev-05, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used

# Required Changes ♻️

- [ ] Kindly make sure you add more content to your readme file, the content should be more descriptive, use [this template](https://github.com/microverseinc/readme-template).
- [ ] Kindly make sure your pull request title is descriptive and covers all activities done in this pull request, also add pull request summary.


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by KayLemba — APPROVED** [review]
Hi @Lizdev-05   👋
Good job working on the project requirements!
There is nothing else to say than its time to merge it. Congratulations 🎉 

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.
 
Optional suggestions
- [OPTIONAL]Please add Proffessional and descriptive PULL REQUEST. Always start your Pull request phases with a capital letter,
This is how you create a PULL Request for example:
Title: Hello Microverse Project
In this milestone I did the following:
1. Add index.html file
2. Add styles.css file
3. Add test.md file to .gitignore
4. Update README.md file 
The above is how you create a descriptive Pull request this should be done in all project you build. 
- [OPTIONAL]Please add a screenshot showcasing your UI  to read README file to make it look more proffessional
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Lizdev-05** [comment]
Thank you so much, I will update the project in line with the corrections

File: .githubworkflows/.githubworkflowslinters.yml.txt (hunk 1 above)
Comment (Lizdev-05) [reply .githubworkflows/.githubworkflowslinters.yml.txt]: Okay, I will check it. Thank you.

File: .gitignore (hunk 2 above)
Comment (Lizdev-05) [reply .gitignore]: Okay, I will do that.

**Pull request merged**